include/aws/common/common.h

```diff
@@ -33,6 +33,11 @@
 #define AWS_STATIC_ASSERT0(cond, msg) typedef char AWS_CONCAT(static_assertion_, msg)[(!!(cond)) * 2 - 1]
 #define AWS_STATIC_ASSERT1(cond, line) AWS_STATIC_ASSERT0(cond, AWS_CONCAT(at_line_, line))
 #define AWS_STATIC_ASSERT(cond) AWS_STATIC_ASSERT1(cond, __LINE__)
+#define AWS_FATAL_ASSERT(cond) if (!cond) {                                                                     \
```

Comment:
Pls wrap `cond` in parens to avoid things like `AWS_FATAL_ASSERT(x == NULL)` becoming `if (!x == NULL)`, which is obviously very different in meaning.